test-builds.sh

```diff
@@ -139,8 +139,8 @@ buildtest() {
 	fi
     else
         if test "${verbose}" != "yes" ; then
-            echo "Build Failed. Last log lines are:"
-            tail -20 ${log}
+            echo "Build Failed. Full log dump:"
+            cat ${log}
```

Comment:
AFAICT, one script execution may have several failures. Marking the huge log start/end would be useful in those cases.

```suggestion
            echo "Build Failed."
            echo "Log start: ${log}"
            cat ${log}
            echo "Log end: ${log}"
```